qiskit/visualization/pass_manager_visualization.py

```diff
@@ -151,8 +153,12 @@ def pass_manager_drawer(pass_manager, filename, style=None, raw=False):
 
         graph.add_subgraph(subgraph)
 
-    if raw and filename:
-        graph.write(filename, format='raw')
+    if raw:
+        if filename:
+            graph.write(filename, format='raw')
+            return None
+        else:
+            raise QiskitError("if format=raw, then a filename is required.")
```

Comment:
Why not use a `qiskit.visualization.exceptions.VisualizationError`?

Reply:
I thought that is not a visualization library error but a error in they way the function is call... But yeah.. fixed in 0911a67b